src/checkout/api_default.go

```diff
@@ -14,41 +14,41 @@ import (
 	_context "context"
 	_nethttp "net/http"
 
-	"github.com/adyen/adyen-go-api-library/v4/src/common"
+	"github.com/adyen/adyen-go-api-library/v5/src/common"
 )
 
 // Checkout Checkout service
 type Checkout common.Service
 
 /*
-GetPaymentLink Retrieve a payment link.
+PaymentLinks Retrieve a payment link.
 Retrieves the payment link details using the payment link &#x60;id&#x60;.
  * @param linkId Unique identifier of the payment link.
  * @param ctxs ..._context.Context - optional, for authentication, logging, cancellation, deadlines, tracing, etc. Passed from http.Request or context.Background().
 @return PaymentLinkResource
 */
-func (a Checkout) GetPaymentLink(linkId string, ctxs ..._context.Context) (PaymentLinkResource, *_nethttp.Response, error) {
+func (a Checkout) GetPaymentLinks(linkId string, ctxs ..._context.Context) (PaymentLinkResource, *_nethttp.Response, error) {
```

Comment:
Shouldn't we keep the old name `GetPaymentLink` since we are retrieving 1 link and not listing multiple? (I understand though the reasoning behind the rename since the resource is named paymentLinks and not paymentLink)

Reply:
Ya, I was just trying to make it consistent, but I dont have a preference here, so if you like it the old way then can change back